homeassistant/components/vartastorage/config_flow.py

```diff
@@ -0,0 +1,107 @@
+"""Config flow for VARTA Storage integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from vartastorage import vartastorage
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {vol.Required("host"): str, vol.Required("port", default=502): int}
+)
```

Comment:
User constants from `homeassistant.const`:

```suggestion
from homeassistant.const import CONF_HOST, CONF_PORT

STEP_USER_DATA_SCHEMA = vol.Schema(
    {vol.Required(CONF_HOST): str, vol.Required(CONF_PORT, default=502): int}
)
```

Reply:
done